Updated instructions for open weather API 3.0

I'm getting 401 errors in the browser console saying this:
```
MMM-WeatherChart.js:119 Error: MMM-WeatherChart: apiKey is invalid
```

Do we need to sign up for a separate API to get access now? 
https://openweathermap.org/api/one-call-3

If so, can you update the readme to mention this?

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ Daily Forecast (max and min temperature, weather icons on the top, rain volume (
 
 MagicMirror module for displaying weather forecasts from [OpenWeather](https://openweathermap.org/). Weather forecasts are displayed by using [Chart.js](https://www.chartjs.org/), an open-source free library for drawing charts.
 
-This module uses the OpenWeather API, which requires an API key. You'll need to subscribe to the [One Call API 3.0](https://openweathermap.org/price) and [limit](https://home.openweathermap.org/subscriptions) the number of calls per day to the free limit so you won't be charged. 
+This module uses the OpenWeather API, which requires an API key. You'll need to subscribe to the [One Call API 3.0](https://openweathermap.org/price) and [limit](https://home.openweathermap.org/subscriptions) the number of calls per day to the free limit so you won't be charged. It can take up to an hour for your API key to start working.
 
 ## Installation
 

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ Daily Forecast (max and min temperature, weather icons on the top, rain volume (
 
 MagicMirror module for displaying weather forecasts from [OpenWeather](https://openweathermap.org/). Weather forecasts are displayed by using [Chart.js](https://www.chartjs.org/), an open-source free library for drawing charts.
 
-This module can work with free OpenWeather API, which only requires to sign up and get an API key.
+This module uses the OpenWeather API, which requires an API key. You'll need to subscribe to the [One Call API 3.0](https://openweathermap.org/price) and [limit](https://home.openweathermap.org/subscriptions) the number of calls per day to the free limit so you won't be charged. 
 
 ## Installation
 

